.github/workflows/docker.yml

```diff
@@ -0,0 +1,67 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches: ['dev']
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image'
+        required: false
+        default: 'latest' # Default tag if none is provided
```

Comment:
this input isn't being used when the tag is set

Reply:
Updated